toml/src/test/kotlin/org/rust/toml/inspections/CargoTomlCrateInspectionTestBase.kt

```diff
@@ -8,24 +8,48 @@ package org.rust.toml.inspections
 import com.intellij.codeInspection.InspectionProfileEntry
 import org.intellij.lang.annotations.Language
 import org.rust.cargo.CargoConstants
+import org.rust.ide.annotator.RsAnnotationTestFixture
 import org.rust.ide.experiments.RsExperiments
 import org.rust.ide.inspections.RsInspectionsTestBase
 import org.rust.openapiext.runWithEnabledFeatures
 import org.rust.toml.crates.local.CargoRegistryCrate
-import org.rust.toml.crates.local.CargoRegistryCrateVersion
 import org.rust.toml.crates.local.withMockedCrates
 import kotlin.reflect.KClass
 
 abstract class CargoTomlCrateInspectionTestBase(
     inspectionClass: KClass<out InspectionProfileEntry>
 ) : RsInspectionsTestBase(inspectionClass) {
-    protected fun doTest(@Language("TOML") code: String, vararg crates: Pair<String, CargoRegistryCrate>) {
-        myFixture.configureByText(CargoConstants.MANIFEST_FILE, code)
+    override fun createAnnotationFixture(): RsAnnotationTestFixture<Unit> =
+        RsAnnotationTestFixture(
+            this,
+            myFixture,
+            inspectionClasses = listOf(inspectionClass),
+            baseFileName = CargoConstants.MANIFEST_FILE
+        )
 
+    private fun runTest(action: () -> Unit, vararg crates: Pair<String, CargoRegistryCrate>) {
```

Comment:
Such function declaration can be more concise:
```runTest(crates: List<Pair<String, CargoRegistryCrate>>, action: () -> Unit)```

It moves function type parameter to the end and removes `vararg` as it doesn't get used here

Reply:
You're right, thanks :)